assets/js/romo/select.js

```diff
@@ -244,27 +240,31 @@ RomoSelect.prototype._setValues = function(newValues) {
   var setValues = newValues.filter(function(value) {
     return currentValues.indexOf(value) === -1;
   });
-  var unsetElems = unsetValues.reduce($.proxy(function(elems, value) {
-    return elems.add(this.elem.find('OPTION[value="'+value+'"]'));
-  }, this), $());
-  var setElems = setValues.reduce($.proxy(function(elems, value) {
-    return elems.add(this.elem.find('OPTION[value="'+value+'"]'));
-  }, this), $());
-
-  unsetElems.removeAttr('selected');
-  unsetElems.prop('selected', false);
-  setElems.attr('selected', 'selected');
-  setElems.prop('selected', true);
+  var unsetElems = unsetValues.map(Romo.proxy(function(value) {
+    return Romo.find(this.elem, 'OPTION[value="'+value+'"]')[0];
+  }, this));
+  var setElems = setValues.reduce(Romo.proxy(function(value) {
+    return Romo.find(this.elem, 'OPTION[value="'+value+'"]')[0];
+  }, this));
+
+  unsetElems.forEach(Romo.proxy(function(unsetElem) {
+    Romo.setAttr(unsetElem, 'selected', '');
```

Comment:
@jcredding does this work for unsetting the DOM attr?  I thought this would result in `<option selected></option>`

Reply:
@jcredding we saw that it doesn't, so we need to add a `.removeAttr` to the base js.  I'll do this in a sep effort.